snuba/consumers/consumer_builder.py

```diff
@@ -181,7 +181,7 @@ def __build_streaming_strategy_factory(self) -> StreamingConsumerStrategyFactory
                 JSONRowEncoder(),
             ),
             max_batch_size=self.max_batch_size,
-            max_batch_time=self.max_batch_time_ms,
+            max_batch_time=self.max_batch_time_ms / 1000.0,
```

Comment:
This diverges from the `BatchProcessingStrategyFactory` but I think most people would reasonably expect a unitless quantity of time would be represented in seconds and not milliseconds (which is why this happened in the first place.)